### PR TITLE
Sabrina's edits

### PR DESCRIPTION
**Pull request by sabrinatkk**
Hi team, 

For my run, I completed the following:
1) emptied content in init.py
2) defined and added plot_lines() function to Colourblind8 package
3) added docstrings to Class and to each function with the package
4) updated code position re: `ax.legend()` to labels' if statement for each function

Please review and comment!

Thanks!

### CODE REVIEW

File: colourblind8/colourblind8.py [hunk 1]
```diff
@@ -1,8 +1,51 @@
...
+        labels : Optional. Labels for each object within the graph.
+        palette (one palette): 'deutera_colours', 'prota_colours','trita_colours'
+        title: Optional. Title for the plot
+        x_lab: Optional. Label for x axis
+        y_lab: Optional. Label for y axis
+        legend_title: Optional. Legend title for legend.
 
+    Returns:
+	
+		an enhanced version of matplotlib plot.
```
Comment (ian-flores) [inline colourblind8/colourblind8.py]: Add the return type of the matplotlib. Also make sure that the documentation follows the basic Numpy Styling. Available here: https://www.numpy.org/devdocs/docs/howto_document.html#docstring-standard

File: colourblind8/colourblind8.py [hunk 2]
```diff
@@ -1,8 +1,51 @@
...
+	
+    """
+	
     def __init__(self):
+		
+		"""
```
Comment (ian-flores) [inline colourblind8/colourblind8.py]: Match indentation in """

File: colourblind8/colourblind8.py [hunk 3]
```diff
@@ -40,13 +179,14 @@ def plot_scatter(self, x, y, alpha = 1,  labels = None, palette = None,
...
                           label = labels[idx])
+			ax.legend(title = legend_title, bbox_to_anchor=(1.01, 1))
         else:
             for idx, values in enumerate(y):
                 ax.scatter(x,
                            values,
                            color = colours[idx],
                            alpha = alpha)
-
+		# if the following parameters are entered, Colourblind8 will apply the theme
```
Comment (ian-flores) [inline colourblind8/colourblind8.py]: I'm not sure this comment is neccessary.

File: colourblind8/colourblind8.py (hunk 1 above)
Comment (sabrinatkk) [reply colourblind8/colourblind8.py]: Thank you for the suggestion! I will reflect this in my next pull request.

File: colourblind8/colourblind8.py (hunk 3 above)
Comment (sabrinatkk) [reply colourblind8/colourblind8.py]: I think it is necessary for viewers to understand the whole flow of our package.

**Pull request merged**